### PR TITLE
Creating Login Page

### PR DESCRIPTION
**Pull request by jchoi25**

### CODE REVIEW

**Comment by jchoi25** [comment]
I have created a sample login page. It currently does not login or remembers the user.

File: login/src/App.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import './App.css';
+import React, { useState } from 'react';
+	
+
+const App = () => {
```
Comment (amit-sawhney) [inline login/src/App.js:5]: Could you add screenshots to the PR?

**Review by amit-sawhney — CHANGES_REQUESTED** [review]

File: login/src/App.js (hunk 1 above)
Comment (jchoi25) [reply login/src/App.js]: I'll do that right now

**Comment by jchoi25** [comment]
I have moved everything into Project. Here is a screenshot of what the login page currently looks like. The Sign In button submits what is typed in the textboxes but does not actually do anything. Remember Me does not do anything as well. Both the username and password are required fields.
screenshot

**Review by amit-sawhney — CHANGES_REQUESTED** [review]
Please remember my instructions. The React app already exists. Right now, you have an entire app called "login" within the project page. You will need to delete this entire app AFTER copying and pasting the code from each file over. 

Let me reiterate:

You will need to copy your code changes, file by file (this will likely only be the App.js file and App.css that is any different). 

Also, can you see if you can center the entire think vertically.

Look into display: flex and align-items: center.

Further note: 

- We won't be having a password or remember me. We don't have a database. Right now, we will just accept a spotify user name and try to pull the data for that Spotify user through the api if possible

**Comment by jchoi25** [comment]
I have removed remember me and the password input. Everything is now aligned in the center.
screenshot

**Review by amit-sawhney — DISMISSED** [review]

**Review by amit-sawhney — APPROVED** [review]

**Pull request merged**